doc/crud.md

```diff
@@ -40,6 +55,10 @@ CDTDatastore *ds = [manager datastoreNamed:@"my_datastore"
 CDTDatastore *ds2 = [manager datastoreNamed:@"other_datastore"
                                       error:&error];
 ```
+```swift
+let ds = try manager.datastoreNamed("my_datastore")
+let ds2 = try manager.datastoreNamed("other_datastore")
```

Comment:
I think it is, the compiler will also remind the user that they need the `catch` if they forgot, but if you or whoever the second reviewer will be thinks not, I can add it in